backend/handlers/createpage.go

```
@@ -44,9 +49,28 @@ func (np *CreatePageHandler) CreatePage(c *gin.Context) {
+	var isTopLevel bool = true
+	if input.ParentID != nil {
+		isTopLevel = false
+	}
+
+	tx, err := np.db.BeginTx(ctx, pgx.TxOptions{})
+	if err != nil {
+		c.Error(api_error.NewInternalServerError("failed to create page", err))
+		return
+	}
+
+	defer tx.Rollback(ctx)
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)